doc/classes/ItemList.xml

```diff
@@ -412,13 +412,15 @@
 				Fired when specified list item has been selected via right mouse clicking.
 				The click position is also provided to allow appropriate popup of context menus
 				at the correct location.
+				[code]allow_rmb_select[/code] must be enabled.
```

Comment:
You can use `[member allow_rmb_select]` to get an hyperlink to the mentioned property.

Reply:
Changed